lib/kvdb/ikvdb.c

```diff
@@ -372,12 +376,38 @@ ikvdb_create(const char *kvdb_home, struct kvdb_cparams *params)
     return err;
 }
 
+static merr_t
+ikvdb_is_pmem_only(const char *kvdb_home, const struct kvdb_meta *meta, bool *pmem_only)
+{
+    merr_t err;
+    int    i;
+
+    INVARIANT(kvdb_home);
+    INVARIANT(meta);
+    INVARIANT(pmem_only);
+
+    *pmem_only = false;
+
+    err = dax_path_is_fsdax(kvdb_home, pmem_only);
+    if (err) {
+         log_err("Failed to determinte the KVDB type for %s", kvdb_home);
+         return err;
+    }
```

Comment:
1. Why do you care if kvdb_home is on a DAX filesystem?
2. Typo in log_err message.

Reply:
kvdb home on fsdax makes pmem as the default media class which permits pmem-only policy, with non-dax filesystem capacity will be the default. I had the same question earlier..

Reply:
(1) is to determine whether the KVDB is of type "pmem-only".

From the RFC:
"A pmem-only KVDB is indicated by creating a KVDB home directory in a DAX file system, in which case it only has the pmem media class, with the default location being KVDB_HOME/pmem (analogous to the default capacity path for a standard KVDB). A pmem-only KVDB supports only the "pmem_only" mclass policy."

Will fix (2), thnx!

Reply:
I forgot about the default path being a subdir of the home dir. :)